docs/install.md

```diff
@@ -1,77 +1,220 @@
+(Pub/Sub topics/subscriptions, output paths, etc) doesn't overlap.
+
+### Persistent Disks
+
+Persistent disks should be the default when processing disks that come from the
+Cloud. The account you run Turbinia as must have access to the Persistent Disks
+that you want to process. If you add a GoogleCloudDisk Evidence type, the worker
+node will attach the disk automatically before it runs its tasks.
+
+### Google Cloud Storage (GCS) Objects
+
+If you have raw disk images from physical machines or otherwise that you want to
+process in the cloud, you can process them directly from GCS. This can be used
+when the images do not originate from VMs/containers that are running in the
+cloud. You can potentially convert raw images to Persistent Disks, but that
+requires processing the image first, and so using GCS is recommended in this
+case. Processing Evidence directly from GCS requires that you use [GCS
+FUSE](https://cloud.google.com/storage/docs/gcs-fuse) and mount your bucket at a
+common path on each worker node. Once your GCS bucket is mounted, you can
+process these images as 'rawdisk' Evidence.
+
+TODO(beamcodeup): Document how to use GoogleCloudDiskRawEmbedded.
+
+# Instructions
+
+## GCP Project (Cloud Pub/Sub, Cloud Function, Cloud Datastore Setup)
+
+**NOTE:** This section is mandatory for Turbinia running on GCP or local
+machines.
+
+*   Create or select a Google Cloud Platform project on the [Google Developers
+    Console](https://console.developers.google.com)
+*   Enable [Cloud
+    Functions](https://console.cloud.google.com/apis/library/cloudfunctions.googleapis.com)
+*   Follow the
+    [instruction](https://cloud.google.com/pubsub/docs/quickstart-console) to:
+    *   Enable [Cloud
+        Pub/Sub](https://console.cloud.google.com/apis/library/pubsub.googleapis.com)
+    *   Create a new Pub/Sub topic and subscription **(pull type)**
+    *   Please take a note of the topic and subscription name for the
+        configuration steps
+*   Enable [Cloud
+    Datastore](https://console.cloud.google.com/apis/api/datastore.googleapis.com)
+    *   Go to Datastore in the cloud console
+    *   Hit the `Create Entity` button
+    *   Select the region that your data should be in. No need to create any
+        Entities after selecting your region
+*   Enable [Cloud Deployment Manager
+    V2](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
+    for managing the GCP via the CLI tool `gclou`
+
+## Local Turbinia
+
+*   If Turbinia will run on local machines, jump to [Core Installation
+    Steps](#core-installation-steps)
+*   Otherwise, follow [GCP Turbinia](#gcp-turbinia)
+
+## GCP Turbinia
+
+The following is a one possible configuration and setup for Turbinia in GCP.
+This is still a rough process and future versions will be containerized.
+
+**NOTE:** GCP Turbinia requires two Google Cloud Engine (GCE) instance types:
+1 server and 1 or more workers
```

Comment:
I would probably say:
"When running Turbinia on GCP, it's recommended that you have at least two VM instances (one for the server, and one or more for the workers).  In a small setup, you can also run both the server and worker on a single instance"

Reply:
done